kokoro/README.md

```diff
@@ -0,0 +1 @@
+CI to run on PR and merge to Master.
```

Comment:
Not important, but up to now we have kept all our build configuration as private files `.github`, `.vscode`, `.cloudbuild`. Is it possible to do the same here with `.kokoro`? Or would that break tooling?

Reply:
That makes sense.  It will not break tooling, but I need to update config and get approval from Kokoro.  I can do that if we prefer to do that or we can do that in a subsequent change once we confirm everything is working.

Reply:
Renamed to `.kokoro`

Reply:
awesome thank you!